scripts/data_collection/get_tweets_from_moe.sh

```diff
@@ -48,50 +49,48 @@ cd ${fiber_home}repo
 
 # Copy to Lisa-Moe shared drive
 echo "$(date -Is) : Copying the iffy list to the Lisa-Moe shared drive."
-rsync -at data/iffy_files/${today}${iffy_filename} truthy@lisa.luddy.indiana.edu:/home/data/${tavern_job}${today}${iffy_filename}
-if [ $? -eq 0 ]; then
-   echo "$(date -Is) : SUCCESS."
+rsync -at data/iffy_files/2023-04-21__iffy_list.txt truthy@lisa.luddy.indiana.edu:/home/data/${tavern_job}${today}${iffy_filename} &> copy_iffy_log.txt
+if grep -q "failed" copy_iffy_log.txt; then
+  echo "$(date -Is) : Copy Faild, Will Exit Now"
```

Comment:
Can we make this more explicit? If we saw this message in the larger master log a year from now it may not be super clear. How about something like...

```
Failed to copy the new iffy file to LISA:${tavern_job}${today}${iffy_filename}
```